docs/configuration/http_conn_man/http_conn_man.rst

```diff
@@ -97,6 +100,40 @@ http_codec_options
   <config_cluster_manager_cluster_http_codec_options>` option. See the comment there about
   disabling HTTP/2 header compression.
 
+.. _config_http_conn_man_http2_settings:
+
+http2_settings
+  *(optional, object)* Additional HTTP/2 codec options that are passed directly to the HTTP/2 codec. 
+  Currently supported settings are:
+
+  hpack_table_size
+    *(optional, integer)* `Maximum Table Size`_ (in octets) that the encoder is permitted to use for
+    the dynamic HPACK table. Valid values range from 0 to 16777216 (2^24) and defaults to 4096,
+    with 0 effectively disables header compression.
+
+    NOTE: A 16MiB (2^24) maximum table size seems to be more than enough.
```

Comment:
this seems wired, the following is not there:
+    NOTE: A 16MiB (2^24) maximum table size seems to be more than enough.

i guess same problem with the test for below:
+      if (http2_settings->hasObject("hpack_table_size") && ret.hpack_table_size_ != 0) {